bigbluebutton-html5/imports/startup/server/userPermissions.js

```diff
@@ -9,6 +9,9 @@ const presenter = {
   subscribePoll: true,
   subscribeAnswers: true,
 
+  // presentation
+  removePresentation: true,
+  sharePresentation: true,
```

Comment:
We will have a feature to "share a presentation" with other people in 2.0 (they will be able to upload it)
It's probably better to change this name to "uploadPresentation" or "showPresentation" depending on the context